src/appleseed.studio/mainwindow/project/tools.cpp

```diff
@@ -261,12 +261,19 @@ void LineEditDoubleSliderAdaptor::adjust_slider(const double new_value)
 
 ForwardColorChangedSignal::ForwardColorChangedSignal(
     QObject*        parent,
+    const QColor&   initial_color,
```

Comment:
It's a detail (and partly a question of taste), but I suggest moving `initial_color` to third position after `widget_name`, because it's a "defining" argument, while `initial_color` is a bit more "peripherical". Make sure to reorder the data members and initializations accordingly.